### ISSUE TITLE
Put the code for brian_mod into a separate h file?

### ISSUE THREAD

**Issue by thesamovar** [brian-team/brian2#597]
At the moment the debug logs are hard to read because of it.

**Comment by mstimberg** [comment]
I agree that this makes things hard to read currently. How about creating a header file for every code object and including that instead of using weave's support code? We could use something like the hash of this file in the comments of the support code to make sure we get a recompile if something in the functions changed.

**Comment by thesamovar** [comment]
Well I'd like to have most of the support code shown because sometimes that's what you're debugging, but the mod stuff is working fine now so could be hidden in a header.

**Comment by thesamovar** [comment]
Actually this isn't really a high priority so I'm removing the 2.0 milestone.

**Comment by mstimberg** [comment]
I agree. It is quite ugly whenever I look at generated code, but users tend to do this less often than we do ;)
